todo_app/trello_items.py

```diff
@@ -0,0 +1,42 @@
+from flask import trello
+import requests
+import json
+
+
+class Item:
+    def __init__(self, id, name, status = 'To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list['name'])
+
+
+def add_items():
+    
+    return trello.get('items')
+
+def get_item(id):
+
+    item = Item.from_trello_card(list)
+
+url = "https://api.trello.com/1/boards/b3L47Acw/lists"
+
+headers = {
+  "Accept": "application/json"
+}
+
+query = {
+  'key': 'APIKey',
+  'token': 'APIToken'
+}
+
+response = requests.request(
+   "GET",
+   url,
+   headers=headers,
+   params=query
+)
+
+print(json.dumps(json.loads(response.text), sort_keys=True, indent=4, separators=(",", ": ")))
```

Comment:
This code all runs exactly once when your app starts, but we want it to run each time the page is loaded so that we show the right info. For example, we want a function that roughly achieves this, that might look like:
```

def get_items():
  # basically the same, but note that I've made it depend on an environment variable
  url = f"https://api.trello.com/1/boards/{os.getenv('TRELLO_BOARD_ID')}/lists"

  headers = {
    "Accept": "application/json"
  }

  # picks up the key/token from environment, also adds the "cards" parameter to load the cards in each list at the same time
  query = {
    'key': os.getenv('TRELLO_KEY'),
    'token': os.getenv('TRELLO_TOKEN'),
    'cards': 'open'
  }

  response = requests.request(
    "GET",
    url,
    headers=headers,
    params=query
  )

  lists = response.json()

  # Probably want to delete this but helpful for debugging
  print(lists)
  # here we want to convert the lists & cards we got back into our "Item" class
  cards = []
  for a_list in lists:
      for a_card in a_list['card']:
          cards.append(Item.from_trello_card(a_card, a_list))
  return cards

```